internal/backends/qt/qt_widgets/palette.rs

```diff
@@ -0,0 +1,176 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+// cSpell: ignore deinit
+
+use i_slint_core::Brush;
+
+use super::*;
+
+cpp! {{
+namespace {
+struct PaletteStyleChangeListener : QWidget {
+    const void *qtStylePalette = nullptr;
+    PaletteStyleChangeListener(const void *qtStylePalette) : qtStylePalette(qtStylePalette) {}
+    bool event(QEvent *event) override {
+        auto ty = event->type();
+        if (ty == QEvent::StyleChange || ty == QEvent::PaletteChange || ty == QEvent::FontChange) {
+            rust!(Slint_qt_style_change_event [qtStylePalette: Pin<&NativePalette> as "const void*"] {
+                qtStylePalette.init_impl();
+            });
+        }
+        return QWidget::event(event);
+    }
+};
+}
+}}
+
+#[repr(C)]
+#[derive(FieldOffsets, SlintElement)]
+#[pin]
+#[pin_drop]
+pub struct NativePalette {
+    pub background: Property<Brush>,
+    pub foreground: Property<Brush>,
+    pub alternate_background: Property<Brush>,
+    pub alternate_foreground: Property<Brush>,
+    pub accent_background: Property<Brush>,
+    pub accent_foreground: Property<Brush>,
+    pub control_background: Property<Brush>,
+    pub control_foreground: Property<Brush>,
+    pub selection_background: Property<Brush>,
```

Comment:
Simliar to the other comment: The documentation documents only `selection`, we discussed `selection-background` and `selection-foreground`, but the implementation here only has `selection-background`. The inconsistency seems... wrong. Or is there something I'm missing?

Reply:
done